ignite/templates/app/files/app/app.go.plush

```diff
@@ -406,6 +445,7 @@ func (app *App) RegisterAPIRoutes(apiSvr *api.Server, apiConfig config.APIConfig
 	docs.RegisterOpenAPIService(Name, apiSvr.Router)
 }
 
+<%= if (!IsChainMinimal) { %>
 // GetIBCKeeper returns the IBC keeper.
 func (app *App) GetIBCKeeper() *ibckeeper.Keeper {
```

Comment:
can we group the `GetIBCKeeper`, `GetSubspace` and `getGovProposalHandlers` functions inside the same if?

eg:
```go
<%= if (!IsChainMinimal) { %>
func (app *App) GetIBCKeeper() *ibckeeper.Keeper {
 ...
}

func (app *App) GetSubspace(moduleName string) paramstypes.Subspace {
...
}

func getGovProposalHandlers() []govclient.ProposalHandler {
...
}
<% } %>
```